### DISCUSSION TITLE
Serve just a single js file

### DISCUSSION THREAD

**Discussion by TheBunyip** [parcel-bundler/parcel#8195] [Q&A] [answered]
I'm using Parcel 2.6 and I'd like to know if what I am trying to achieve is possible...

I want to be able to serve `src/index.ts` for browsers to consume from `dist/my-new-name-for-index.js`.

Note that I do not have (or want) an HTML file.  Since I don't have an HTML file, I cannot specify that `my-new-name-for-index.js` should be treated as a module.  So I want Parcel to generate that JS file with all the `import/require` calls remove (and inlined). 

If I just run `parcel serve` it makes a `dist/index.js` which looks right, but I want the name to be `my-new-name-for-index.js`

I have tried various target configurations in `package.json` but I can't seem to configure it right.  Am I trying to do something beyond what Parcel can do?

**Comment by mischnic** [comment, marked as answer]
```json
{
	"scripts": {
		"start": "parcel src/index.ts --target default"
		"build": "parcel build src/index.ts"
	},
	"default": "dist/my-new-name-for-index.js"
	"targets": {
		"default": {}
	}
}
```

**Reply by TheBunyip** [reply]
Great, that works!  (And thanks for the speedy reply.)

That was a config I hadn't tried.  I don't think I'd have ever worked out the need for this part:

```json
"targets": {
  "default": {}
}
```
